<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WmiWidget</class>
 <widget class="QWidget" name="WmiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>143</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblHost">
       <property name="text">
        <string>Host </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1">
       <item>
        <widget class="QComboBox" name="cmbHost">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>localhost</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblNameSpace">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbNameSpace">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>root\cimv2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblQuery">
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbQuery">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnQuery">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../qHostMonitor.qrc">
           <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblAlertIf">
       <property name="text">
        <string>Alert if</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbAlertIf">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>any HandleCount</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>average</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>total</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>row count</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbAlertSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
         <item>
          <property name="text">
           <string>is &lt; than</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is &gt; than</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is = to </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>is &lt;&gt; from</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>increases by ..</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>decreases by ..</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>changes by ..</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>increases by ..(%)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>decreases by ..(%)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>changes by ..(%)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>increases /sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>decreases /sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>changes /sec</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnAlertIf">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblIfInstanceNotAvailable">
       <property name="text">
        <string>If Instance not available, set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbInstance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Unknown status</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bad status</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ok status</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
